RLTest/__main__.py

```diff
@@ -327,6 +331,10 @@ def __init__(self):
             print(Colors.Green('RLTest version {}'.format(__version__)))
             sys.exit(0)
 
+        if self.args.redis_port not in range(0, pow(2, 16)):
```

Comment:
port 0 is reserved and technically unavailable. If you're going to go with way then:
```range(1, pow(2,16))```

Reply:
0 is a valid value implying port randomization - I wouldn't like to exclude it here.
Let's not complicate this too much - we don't generally check port numbers in RLTest.